sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/part-time-calculation-process.ts

```diff
@@ -57,6 +59,7 @@ export class PartTimeCalculationProcess extends ECertCalculationProcess {
       this.validateDisbursementPartTimeStep,
       this.applyOverawardsDeductionsStep,
       this.calculateEffectiveValueStep,
+      this.validateCSLPLifetimeMaximumCheckStep,
```

Comment:
@guru-aot  FMU, can you please tell why this step is placed after the `calculateEffectiveValueStep` .

Reply:
It should be below the validateDisbursementPartTimeStep, nicely noted.